src/hotspot/share/utilities/compilerWarnings.hpp

```diff
@@ -86,25 +86,58 @@
 #define PRAGMA_ZERO_AS_NULL_POINTER_CONSTANT_IGNORED
 #endif
 
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
This doesn't adhere to the style of the file AFAICS and is unnecessary, can we remove it?